modules/apps/knowledge-base/knowledge-base-test/src/testIntegration/java/com/liferay/knowledge/base/internal/upgrade/v4_4_0/test/RSSConfigurationUpgradeProcessTest.java

```diff
@@ -0,0 +1,185 @@
+ * details.
+ */
+
+package com.liferay.knowledge.base.internal.upgrade.v4_4_0.test;
+
+import com.liferay.arquillian.extension.junit.bridge.junit.Arquillian;
+import com.liferay.knowledge.base.configuration.KBGroupServiceConfiguration;
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.kernel.test.rule.AggregateTestRule;
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+import com.liferay.portal.kernel.upgrade.UpgradeStep;
+import com.liferay.portal.kernel.util.HashMapDictionaryBuilder;
+import com.liferay.portal.test.log.LogCapture;
+import com.liferay.portal.test.log.LoggerTestUtil;
+import com.liferay.portal.test.rule.Inject;
+import com.liferay.portal.test.rule.LiferayIntegrationTestRule;
+import com.liferay.portal.test.rule.PermissionCheckerMethodTestRule;
+import com.liferay.portal.upgrade.registry.UpgradeStepRegistrator;
+
+import java.util.Dictionary;
+import java.util.Objects;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import org.osgi.framework.Constants;
+import org.osgi.service.cm.Configuration;
+import org.osgi.service.cm.ConfigurationAdmin;
+
+/**
+ * @author Marco Galluzzi
+ */
+@RunWith(Arquillian.class)
+public class RSSConfigurationUpgradeProcessTest {
+
+	@ClassRule
+	@Rule
+	public static final AggregateTestRule aggregateTestRule =
+		new AggregateTestRule(
+			new LiferayIntegrationTestRule(),
+			PermissionCheckerMethodTestRule.INSTANCE);
+
+	@Before
+	public void setUp() throws Exception {
+		_configuration = _getConfiguration();
+
+		if (_configuration == null) {
+			_configuration = _configurationAdmin.getConfiguration(
+				_SERVICE_PID, StringPool.QUESTION);
+			_originalProperties = null;
+		}
+		else {
+			_originalProperties = _configuration.getProperties();
+		}
+
+		_configuration.update(
+			HashMapDictionaryBuilder.putAll(
+				_originalProperties
+			).put(
+				"rssDelta", Integer.valueOf("20")
+			).put(
+				"rssFormat", "atom10"
+			).build());
+	}
+
+	@After
+	public void tearDown() throws Exception {
+		if (_originalProperties == null) {
+			_configuration.delete();
+		}
+		else {
+			_configuration.update(_originalProperties);
+		}
+	}
+
+	@Test
+	public void testRSSConfigurationUpgradeProcessWithExistingConfiguration()
+		throws Exception {
+
+		_checkPropertiesBefore(_configuration.getProperties());
+
+		_runUpgrade();
+
+		_configuration = _getConfiguration();
+
+		_checkPropertiesAfter(_configuration.getProperties());
+	}
+
+	private void _checkPropertiesAfter(Dictionary<String, Object> properties) {
+		Assert.assertTrue(
+			"The property 'rssDelta' should be of type String",
+			properties.get("rssDelta") instanceof String);
+		Assert.assertNull(
+			"The property 'rssFormat' should not exist",
+			properties.get("rssFormat"));
+	}
+
+	private void _checkPropertiesBefore(Dictionary<String, Object> properties) {
```

Comment:
In tests, methods that check assertions should be named `_assert*`, not `_check*`.

Reply:
Done, thanks for the info